cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -228,6 +228,8 @@ handleSimpleNode p NodeCLIArguments{..}
 
       varTip <- atomically $ newTVar GenesisPoint
       epochInfo <- newEpochInfo $ nodeEpochSize (Proxy @blk)
+      -- Database filepath
+      path <- makeAbsolute ccDBPath
```

Comment:
I think that `makeAbsolute` is not enough to resolve any `..` inside the path, you'll need `canonicalizePath`:

```
$ pwd
/home/thomas
$ ghci
> import System.Directory
> makeAbsolute "../../home/thomas"
"/home/thomas/../../home/thomas"
> canonicalizePath =<< makeAbsolute "../../home/thomas"
"/home/thomas"
```